package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "Spotty",
+  "version": "1.0.0",
+  "description": "spotty",
+  "main": "src/js/index.js",
+  "scripts": {
+    "build": "webpack --mode development",
+    "start": "webpack-dev-server --mode development"
+  },
+  "keywords": [],
+  "license": "ISC",
+  "devDependencies": {
+    "@material/ripple": "^0.40.0",
+    "autoprefixer": "^9.1.5",
+    "css-loader": "^1.0.0",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
I would pick MiniCssExtractPlugin instead of this to minify css as webpack docs says. It is still beta for webpack 4